doc/packages/krita.section.md

```diff
@@ -0,0 +1,27 @@
+# Krita {#sec-krita}
+
+## Installing binary plugins {#krita-install-plugins}
+
+You can choose what plugins are added to Krita by overriding the
+`plugins` attribute.
+
+If you want to add plugins instead of replacing you can read the
+list of default plugins at via `pkgs.krita.plugins`:
+
+```nix
+(pkgs.krita.override {
+    plugins = pkgs.krita.plugins ++ [
+        pkgs.krita-plugin-gmic
+        your-other-plugin
+    ];
```

Comment:
I think this should simply be `plugins = [ pkgs.krita-plugin-gmic your-other-plugin ];`?

Reply:
Yeah, the example is too long :+1: